jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/io/FrameFlusher.java

```diff
@@ -113,6 +114,7 @@ private Action flush()
         {
             if (!BufferUtil.isEmpty(aggregate))
             {
+                aggregate.flip();
```

Comment:
The BufferUtil class has unintended side effects for the various flip methods.
We used to use BufferUtil here, but changed to flip to avoid those side effects.